x-pack/filebeat/module/cisco/amp/config/config.yml

```diff
@@ -0,0 +1,71 @@
+{{ if eq .input "httpjson" }}
+
+type: httpjson
+config_version: "2"
+interval: {{ .interval }}
+
+auth.basic.user: {{ .client_id }}
+auth.basic.password: {{ .api_key }}
+
+request.url: {{ .url }}
+request.method: GET
+request.timeout: {{ .request_timeout }}
+request.ssl: {{ .ssl | tojson }}
```

Comment:
Should this be surrounded with `{{ if .ssl }}` since `ssl` has no default in the manifest? I figured you'd get an error if ssl is not used in the config and you are using the httpjson input.

Reply:
Same goes for client_id and api_key.